mods/ts/rules/aircraft.yaml

```diff
@@ -144,12 +146,19 @@ ORCATRAN:
 		Cost: 1200
 	Tooltip:
 		Name: Orca Transport
+	Buildable:
```

Comment:
This is unrelated to takeoff and landing sounds, so needs to be at least in its own commit.